### PR TITLE
Feature/139920 wss support

### PR DESCRIPTION
**Pull request by vananyev**

### CODE REVIEW

**Review by dmitriyisaev — CHANGES_REQUESTED** [review]
Please change target branch to dev

**Review by dmitriyisaev — COMMENTED** [review]
Do we have WSS test? I Mean that after https request for websocket connection we do connect vis wss?

**Comment by vananyev** [comment]
@dmitriyisaev , right now the test does not contain wss connection. It will be in the next test. Could you check that this kind of test using is suitable?

**Comment by vananyev** [comment]
@dmitriyisaev , i have changed the schema in the test, and debug it. Actually it uses the ssl security in the netty during data-exchange. So could you check does this test enough or we need more?

File: conf/application.conf [hunk 1]
```diff
@@ -81,6 +81,9 @@ play {
 
 	# Setting up https settings and values
 	# For more options check https://www.playframework.com/documentation/2.6.x/SettingsAkkaHttp
+	# The manual for creating ROOT sertificate is: https://lightbend.github.io/ssl-config/WSQuickStart.html
+	# The property of the generated selfsign key is: CN=Admin, OU=IT, O=Acuitybrands, L=Athlas, ST=CA, C=US
+	# Extra properties are: Alias: scala-broker, Password: acuitybrands
```
Comment (dmitriyisaev) [inline conf/application.conf:86]: Thanks, very helpful.

**Review by dmitriyisaev — APPROVED** [review]

**Comment by dmitriyisaev** [comment]
Looks fine for me

**Pull request merged**